packages/react-components/react-combobox/stories/src/Dropdown/DropdownTruncation.stories.tsx

```diff
@@ -35,14 +40,15 @@ export const TruncatedValue = (props: Partial<DropdownProps>) => {
     'Fox',
     'Hamster',
     'Snake',
+    'SuperLongName_123456789@soyouwantalongurl.com',
```

Comment:
I think we should avoid adding URLs we don't own to our docs.

```suggestion
    'SuperLongName_123456789_SomeMoreStuffToMakeItLonger@fluentui.dev',
```

Reply:
oh good call :D